r3bdata/ttt10Data/R3BTTTXHitData.cxx

```diff
@@ -0,0 +1,50 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2024 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+// -------------------------------------------------------------------------
+// -----                  R3BTTTXHitData source file                   -----
+// -------------------------------------------------------------------------
+
+#include "R3BTTTXHitData.h"
+#include <fmt/core.h>
+
+R3BTTTXHitData::R3BTTTXHitData(uint8_t detID, double xpos, double energy, double angle, double charge, double time)
```

Comment:
Probably we don't need detID. I am thinking of using this Hit Level to store the angle deduced from two (or more) detectors. So the detID doesn't make sense at this stage.

Reply:
Ok, but the angle should be given only by the TTT10 detectors, the combination with other detectors for tracking is part of tracking tasks

Reply:
Right. The angle here will be just ratios x-diff / z-diff between two detectors and will not give precise angles but still probably good to try to calculate...